feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-single-selection-base.component.ts

```diff
@@ -137,6 +137,23 @@ export abstract class ConfiguratorAttributeSingleSelectionBaseComponent extends
     };
   }
 
+  /**
+   * Extract corresponding value price formula parameters
+   *
+   * @param {Configurator.Value} value - Configurator value
+   * @return {ConfiguratorPriceComponentOptions} - New price formula
+   */
+  extractValuePriceFormulaParameters(
+    value?: Configurator.Value
+  ): ConfiguratorPriceComponentOptions | undefined {
+    if (value) {
+      return {
+        price: value?.valuePrice,
```

Comment:
```suggestion
        price: value.valuePrice,
```
value is defined!

Reply:
Yes, I know. Still the chaining operator is not needed

Reply:
Done!